resources/lang/fr/actions.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+return [
+    'actions'           => 'Actions',
+    'attach'            => 'Attacher',
+    'add'               => 'Ajouter',
+    'create'            => 'Créer',
+    'add'               => 'Ajouter',
+    'view'              => 'vue',
+    'edit'              => 'Éditer',
+    'delete'            => 'effacer',
+    'cancel'            => 'Annuler',
+    'save'              => 'enregistrer',
+    'close_search'      => 'fermer la recherche',
+    'close_practice'    => 'pratique étroite',
+    'add_lost_person'   => 'ajouter une personne perdue',
+    'add_version'       => 'ajouter une version',
+    'reopen'            => 'rouvrir',
+    'go_back'           => 'retourner',
+    'load'              => 'charge',
+    'closing'           => 'fermeture',
+    'no'                => 'Non',
+    'yes'               => 'Oui',
+    'filter_by_status'  => 'filtrer par statut',
+    'filter_by_dates'   => 'filtrer par dates',
+    'filter_by_region'  => 'filtrer par région',
+    'filter_by_village' => 'filtrer par village',
+    'filter_by_groups'  => 'filtrer par groupes',
+    'filter_by_types'   => 'filtrer par type',
```

Comment:
First letter has to be uppercase in all these strings. Please look at the English version to follow the same style in uppercase letters.